### PR TITLE
Add support and help page

### PR DESCRIPTION
**Pull request by flickz**
Page route `/help` , for contact page `/contact/` and  confirmation page `/contact/success`

### CODE REVIEW

**Comment by flickz** [comment]
@znarf I've updated the code to follow the foundation contact-us pattern. We might need to add the following environment variable to the frontend if not yet there.

```
 MAILGUN_USER
 MAILGUN_PASSWORD
```

**Comment by znarf** [comment]
Feedback.

1) Toolbar is acting a bit weird, too many options. Can we remove some so everything fits without scroll?

Screenshot 2021-12-13 at 19 49 56

_More coming._

File: rewrites.js [hunk 1]
```diff
@@ -309,10 +309,18 @@ exports.REWRITES = [
     source: '/how-it-works',
     destination: '/how-it-works',
   },
+  {
+    source: '/api/support',
+    destination: '/api/support',
+  },
```
Comment (znarf) [inline rewrites.js:315]: Can we remove this?

File: lib/graphql/schemaV2.graphql [hunk 2]
```diff
@@ -7278,6 +7278,11 @@ type Mutation {
     """
     virtualCard: VirtualCardReferenceInput!
   ): Boolean
+
+  """
+  Send help and support message
+  """
+  sendMessage(name: String!, email: String!, topic: String!, link: String, message: String!): SupportResponse!
```
Comment (znarf) [inline lib/graphql/schemaV2.graphql:7285]: This should be removed right? `npm run graphql:update` with main.

File: rewrites.js (hunk 1 above)
Comment (flickz) [reply rewrites.js:315]: @znarf This was the only way I could get it working. It did not work without this.

**Pull request merged**